### ISSUE TITLE
:link: Ajouter lien cliquable pour ouvrir la page du domaine sur domaine.nc :bulb: 

### ISSUE THREAD

**Issue by adriens** [lschaeffer313/domaine-nc-mobile#102]

**Comment by adriens** [comment]
Vous en pensez quoi ?

**Comment by HakumenNC** [comment]
Bien pratique et "pas cher" à implémenter 🔥

**Comment by adriens** [comment]
Yep, faudra voir où positionner ces liens car on en a deux  (à priori, j'en vois pas plus à ce stade): 

- url sur domaine.nc
- url du site en question

**Comment by adriens** [comment]
> ça rejoint un peu l'idée du https://github.com/lschaeffer313/domaine-nc-mobile/issues/101

Complétement :+1:

**Comment by lschaeffer313** [comment]
Dans un premier temps je peux proposer quelque chose comme ça.
<img width="457" alt="image" src="https://user-images.githubusercontent.com/28922072/219004618-9d4ee348-5491-481d-b67e-00bbfc24dd49.png">

**Comment by lschaeffer313** [comment]
Mais il faudrait voir un moyen plus "moderne" d'afficher les liens

**Comment by adriens** [comment]
Ouep, ça jure u npeu pour l'instant.

Sur la partie lien, je verrai bien un truc du genre :link: 

et pou r chque lien un tag pour chaque url dont le nom serait  `www` et `domaine.nc` genre... tu vois ce que jeux dire ?

**Comment by adriens** [comment]
Meême plus la peine d'écrire "LIens" du coup

**Comment by adriens** [comment]
:hand:  Lors de la [démo d'unboxing](https://youtu.be/wD2mH2LMrP0?t=406), je me suis rendu compte que cette feature manquait vraiment
